ledger/sandbox-on-x/src/test/suite/scala/com/daml/ledger/sandbox/bridge/validate/SequenceSpec.scala

```diff
@@ -123,37 +123,13 @@ class SequenceSpec extends AnyFlatSpec with MockitoSugar with Matchers with Argu
     )
   }
 
-  it should "reject a transaction on unallocated transaction informees" in new TestContext {
+  it should "fail on unallocated transaction informees" in new TestContext {
     val Seq((offset, update)) = sequence(input(txWithUnallocatedParty))
     offset shouldBe toOffset(1L)
     assertCommandRejected(update, "Parties not known on ledger: [new-guy]")
   }
 
-  it should "reject a transaction if there is no ledger configuration" in new TestContext {
```

Comment:
Not sure why you needed to delete this test

Reply:
right, by mistake. My bad. Added it back.